Source/SwiftLintCore/Models/RuleConfigurationDescription.swift

```diff
@@ -391,9 +429,14 @@ public struct ConfigurationElement<T: AcceptableByConfigurationElement & Equatab
     /// parent configuration in this specific case.
     ///
     /// - Parameter value: Value to be wrapped.
-    public init(wrappedValue value: T) where T: InlinableOptionType {
+    public convenience init(wrappedValue value: T) where T: InlinableOptionType {
         self.init(wrappedValue: value, key: "")
     }
+
+    /// Run operations to validate and modify the parsed value.
+    public func performAfterParseOperations() throws {
```

Comment:
I guess this is what drives the switch from struct to class? Would be nice to find a way back to using a value type but not a big deal.

Reply:
Yeah, it hurt when I changed `struct` to `class`. I cannot think of a way to modify the `@ConfigurationElement` instance itself if it's a value type as this involves some kind of "getter" that returns a copy-on-write object.